gateway/cli/src/main.rs

```diff
@@ -91,9 +96,54 @@ pub enum Commands {
 
         #[clap(long)]
         network: Option<bitcoin::Network>,
+
+        /// Format <federation id>,<base msat>,<proportional to millionths part>
+        #[clap(long)]
+        per_federation_routing_fees: Option<Vec<FederationIdAndRoutingFeesParameter>>,
```

Comment:
Do we even need to add this? Just for simplicity we could just use the default federation routing fees when connecting a new federation

Reply:
Good one, less moving parts is better, I'll remove it

Reply:
let's remove. I think it's  cheap to connect federation and then configure fees and any other params once connected. This is also intuitive for non CLI ux people will build for gateway